spec/front_desk_spec.rb

```diff
@@ -0,0 +1,398 @@
+require_relative "spec_helper"
+
+module Hotel
+  describe "class FrontDesk" do
+    def build_front_desk
+      return Hotel::FrontDesk.new
+    end
+
+    describe "all arrays" do
```

Comment:
Not the best name for this test! What if these values grow to not be arrays in the future? This is testing that the initial values are correct